command/meta.go

```diff
@@ -50,6 +50,11 @@ type Meta struct {
 	// Override certain behavior for tests within this package
 	testingOverrides *testingOverrides
 
+	// Override the set of provider plugin SHA256 digests we expect.
+	// If this is nil we will instead read from the provider lock file
+	// when setting up ContextOpts.
+	forceProviderSHA256s map[string][]byte
```

Comment:
This is also what we would use to override to use hashes stored in the plan, once that is implemented. (I had a prototype working with this but decided to treat that as a separate bit of work since this is big enough already.)